presto-docs/src/main/sphinx/release/release-0.223.rst

```diff
@@ -0,0 +1,24 @@
+=============
+Release 0.223
+=============
+
+General Changes
+---------------
+
+* Fix a bug where a connector may not optimize query plan if the corresponding
+  plugin is installed after the server is started.
+* Fix incorrect optimizations that might remove necessary CAST (:pr:`13117`).
+* Add support for multibyte characters in :func:`date_format`.
+* Add support for using long polling for task information updates to improve
+  coordinator CPU utiliation. ``experimental.task.info-update-refresh-max-wait``
+  is the configuration property to enable this.
+
+Web UI Changes
+--------------
+
+* Fix splits timeline view.
```

Comment:
What was the problem?  It didn't work entirely?

Reply:
I am not sure @rschlussel added this

Reply:
i believe it was just totally broken.  This was @mayankgarg1990's change.

Reply:
Yes - due to a missing JS file - it was completely broken